phpstan-baseline.neon

```diff
@@ -0,0 +1,30 @@
+parameters:
+	ignoreErrors:
+		-
+			message: "#^Call to an undefined method Pimcore\\\\Model\\\\Dao\\\\AbstractDao\\:\\:getById\\(\\)\\.$#"
+			count: 1
+			path: src/FavoriteOutputDefinition.php
+		-
+			message: "#^Call to an undefined method Pimcore\\\\Model\\\\Dao\\\\AbstractDao\\:\\:save\\(\\)\\.$#"
+			count: 1
+			path: src/FavoriteOutputDefinition.php
+		-
+			message: "#^Call to an undefined method Pimcore\\\\Model\\\\Dao\\\\AbstractDao\\:\\:delete\\(\\)\\.$#"
```

Comment:
I am not sure but these should be all fixed by adding in phpdoc  `@method Web2PrintToolsBundle\FavoriteOutputDefinition\Dao delete()`  or so

Reply:
unresolving: it has to be added to all the other as well eg. `getById`,`save`. You can put in on top before the `class`

Reply:
Well, the fix you suggested was not working properly, So I added @method and @property type hints in Dao and Model classes in order to reference each other and the errors are gone